cmake/modules/AddSwift.cmake

```diff
@@ -1109,7 +1110,7 @@ function(_add_swift_library_single target name)
       COMPILE_FLAGS " ${c_compile_flags}")
   set_property(TARGET "${target}" APPEND_STRING PROPERTY
       LINK_FLAGS " ${link_flags}")
-  swift_target_link_search_directories(${target} ${library_search_directories})
+  swift_target_link_search_directories("${target}" "${library_search_directories}")
```

Comment:
I saw target quoted in multiple calls to `add_dependencies`, `add_custom_command_target`, `add_library`, `_set_target_prefix_and_suffix`, `target_include_directories`, `set_target_properties` among others. But you're right, there's no specific need for it as targets are not supposed to handle spaces, and multiple pieces of code lack of support of multi-target handling on the same call.